geyser-plugin-manager/src/geyser_plugin_manager.rs

```diff
@@ -73,3 +64,47 @@ impl GeyserPluginManager {
         false
     }
 }
+
```

Comment:
Why adding new code commented out? Please mark the PR in draft mode and turn it back on if it is ready for review?

Reply:
Oops, forgot to clean this up. All the code that isn't commented is what I want a review on.